libraries/math/src/vector3/coreHelpers.ts

```diff
@@ -159,6 +159,66 @@ export class Vector3Utils {
         const tb = Math.sin(t * theta) / sinTheta;
         return Vector3Utils.add(Vector3Utils.scale(a, ta), Vector3Utils.scale(b, tb));
     }
+    /**
+     * multiply
+     * 
+     * Multiplies two vectors together
+     */
+    static multiply(a: Vector3, b: Vector3): Vector3 {
+        return { 
+            x: a.x * b.x, 
+            y: a.y * b.y, 
+            z: a.z * b.z 
+        };
+    }
+
+    /**
+     * rotateX
+     * 
+     * Rotates the vector around the x axis
+     * @param a Angle in radians
```

Comment:
I ran the pipeline for the PR and you have a few lint errors to fix. Specifically, you need params with a dash after the parameter name. If you run `npm run lint` locally you'll get the full errors to fix (including the need to run prettier which you can do with the `--fix` option).

Reply:
Sorry about that
`npm run lint` fails on my machine on the prettier stage.
`Error: Command failed: C:\Program Files\nodejs\node.exe C:\Users\aiaub\Documents\VSCode\minecraft-scripting-libraries\libraries\math\node_modules\prettier\bin-prettier.js --check src/**/*.ts`
I was hoping the vscode extension would catch everything, but I'll try to figure out what's wrong.